sdk/communication/Azure.Communication.NetworkTraversal/samples/Sample1_CommunicationRelayClient.md

```diff
@@ -40,7 +40,7 @@ A configuration is returned for the user. Each configuration consists of a URL f
 Every relay configuration has an expiry date and time stamped on it, indicating when the set of TURN credentials will no longer be accepted. Once the credentials have expired, you can renew the token by calling the same method.
 
 ```C# Snippet:GetRelayConfiguration
-Response<CommunicationRelayConfiguration> relayConfiguration = client.GetRelayConfiguration(user);
+Response<CommunicationRelayConfiguration> relayConfiguration = client.GetRelayConfiguration(new GetRelayConfigurationOptions(user));
```

Comment:
For the basic scenario I would recommend to use the version without the options, so just `client.GetRelayConfiguration()`. Same for the async snippet.